src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,42 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represent an Event that has additional LocalDateTime field and extends Task class.
+ */
+public class Event extends Task {
+    private LocalDateTime eventDateTime;
+    private String at;
```

Comment:
I like how you are adhering to encapsulation principle by making your attributes private. I had some trouble deciphering what does the at variable represent without reading the rest of the code base though. Would it be better to refactor it into "atDateTime"? 

*this could apply similarly to the code for the class deadline as well :)